python/pyfeelpp/feelpp/__init__.py.in

```diff
@@ -32,13 +32,24 @@ def readCfg(filename):
 
     Yields:
         ConfigParser: 
-    """    
+    """
+    from collections import OrderedDict
     from configparser import ConfigParser
-    config = ConfigParser()
     # use this function to add a 
     def FeelppHead(fp):
         yield '[feelpp]\n'
         yield from fp
+    class MultiOrderedDict(OrderedDict):
+        def __setitem__(self, key, value):
+            if key in self:
+                if isinstance(value, list):
+                    self[key].extend(value)
+                    return
+                elif isinstance(value,str):
+                    return # ignore conversion list to string (line 554)
+            super(MultiOrderedDict, self).__setitem__(key, value)
+
+    config = ConfigParser.RawConfigParser(dict_type=MultiOrderedDict, strict=False)
```

Comment:
@prudhomm it seems that `RawConfigParser` is not a member of the class `ConfigParser`, but a class of the library `configparser` (cf [the documentation](https://docs.python.org/3.10/library/configparser.html#rawconfigparser-objects)).
When I run the test locally, I get this error :
```sh
3: >       config = ConfigParser.RawConfigParser(dict_type=MultiOrderedDict, strict=False)
3: E       AttributeError: type object 'ConfigParser' has no attribute 'RawConfigParser'
3: 
3: /data/home/saigre/develop/install/default/lib/python3.10/site-packages/feelpp/core/__init__.py:54: AttributeError
3: =========================== short test summary info ============================
3: FAILED ../../../../../python/pyfeelpp/tests/test_core.py::test_config_parser - AttributeError: type object 'ConfigParser' has no attribute 'RawConfigParser'
```
Oddly, in the PR the test appear to have passed, but they actually failed : https://github.com/feelpp/feelpp/actions/runs/10657011721/job/29546848919